cdap-client/src/main/java/co/cask/cdap/client/MetadataClient.java

```diff
@@ -145,25 +147,25 @@ public MetadataClient(ClientConfig config) {
 
   /**
    * @param id the entity for which to retrieve metadata tags
+   * @param scope the {@link MetadataScope} to retrieve the tags from. If unspecified, this method retrieves
+   *              tags from both {@link MetadataScope#SYSTEM} and {@link MetadataScope#USER}
    * @return The metadata tags for the entity.
    */
-  public Set<String> getTags(Id id)
+  public Set<String> getTags(Id id, MetadataScope scope)
```

Comment:
What do you mean 'if unspecified'. If null is passed in? If so, mark it nullable?